src/braket/circuits/circuit.py

```diff
@@ -635,6 +640,9 @@ def add_verbatim_box(
         if verbatim_circuit.result_types:
             raise ValueError("Verbatim subcircuit is not measured and cannot have result types")
 
+        if verbatim_circuit._measure_targets:
+            raise ValueError("Verbatim subcircuit is not measured.")
```

Comment:
I see that this error is derived from the one above, but it's a bit confusing to me personally

Reply:
Would something like `"Cannot measure a subcircuit inside a verbatim box."` be better?